src/mbgl/annotation/annotation_tile.cpp

```diff
@@ -47,12 +47,9 @@ FeatureType AnnotationTileFeature::getType() const {
     return data->type;
 }
 
-optional<Value> AnnotationTileFeature::getValue(const std::string& key) const {
+Value AnnotationTileFeature::getValue(const std::string& key) const {
     auto it = data->properties.find(key);
-    if (it != data->properties.end()) {
-        return optional<Value>(it->second);
-    }
-    return optional<Value>();
+    return it != data->properties.cend() ? it->second : NullValue();
```

Comment:
No need to use `cend`; The `const` overload of `end` will automatically be used since this function is `const`.